components/portal/messages/controllers.js

```diff
@@ -688,6 +688,8 @@ define(['angular'], function(angular) {
             $rootScope.addPortletToHome(fName);
             actionButton.label = 'On your home';
             actionButton.disabled = true;
+          } else {
+            $window.open(actionButton.url, '_blank');
```

Comment:
does `window.open` automatically work like a link with `rel="noopener noreferrer"`?

Reply:
Will the URL always be external-to-the-portal, or might it be sensible to mascot-announce a path within the portal that's not a widget to add to home?

Reply:
I explicitly added the re=noopener noreferrer and logic to distinguish between internal and external links.